scripts/west_commands/run_common.py

```diff
@@ -260,7 +260,7 @@ def load_cmake_cache(build_dir, args):
 def rebuild(command, build_dir, args):
     _banner(f'west {command.name}: rebuilding')
     try:
-        zcmake.run_build(build_dir)
+        zcmake.run_build(build_dir, target="west_flash_depends")
```

Comment:
This is shared code called by all the runner related commands, which includes things like debug, debugserver, and attach. It looks to me like this is assuming that it's only called for flash, even though the changes to cmake/flash/CMakeLists.txt do produce a separate dependency target for each command.

Shouldn't this be something like `target=f"west_{command.name}_depends"` instead, then?